server/sonar-server/src/main/java/org/sonar/server/user/ThreadLocalUserSession.java

```diff
@@ -87,23 +87,38 @@ public Locale locale() {
   }
 
   @Override
+  public boolean isRoot() {
+    return get().isRoot();
+  }
+
+  @Override
+  public UserSession checkIsRoot() {
+    get().checkIsRoot();
+    return this;
+  }
+
+  @Override
   public UserSession checkLoggedIn() {
-    return get().checkLoggedIn();
+    get().checkLoggedIn();
+    return this;
   }
 
   @Override
   public UserSession checkPermission(String globalPermission) {
-    return get().checkPermission(globalPermission);
+    get().checkPermission(globalPermission);
+    return this;
   }
 
   @Override
   public UserSession checkGlobalPermission(String globalPermission) {
-    return get().checkGlobalPermission(globalPermission);
+    get().checkGlobalPermission(globalPermission);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "checkGlobalPermission"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)